x-pack/test/alerting_api_integration/security_and_spaces/group3/tests/maintenance_window/external/create_maintenance_window.ts

```diff
@@ -94,15 +98,34 @@ export default function createMaintenanceWindowTests({ getService }: FtrProvider
       });
     }
 
-    it('should throw if creating maintenance window with invalid scoped query', async () => {
+    it('should throw if creating maintenance window with invalid kql', async () => {
       await supertest
         .post(`${getUrlPrefix('space1')}/api/alerting/maintenance_window`)
         .set('kbn-xsrf', 'foo')
         .send({
-          ...createParams,
+          ...createRequestBody,
           scope: {
-            query: {
-              kql: 'invalid_kql:',
+            alerting: {
+              query: {
+                kql: 'invalid_kql:',
+                solutionId: 'securitySolution',
```

Comment:
This is not supported any more.

Reply:
I thought i had removed all of them, thanks!